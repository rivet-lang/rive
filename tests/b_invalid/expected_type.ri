func my_fn() { }

func x(a: my_fn) {}

func main() { }
