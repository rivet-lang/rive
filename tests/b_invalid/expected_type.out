tests/b_invalid/expected_type.ri:3:11: error: expected type, found function
    3 | func x(a: my_fn) {}
      |           ^
rivetc: error: could not compile module `expected_type`, aborting due to previous error